db/migrate/20151114185422_add_default_value_for_order_status.rb

```diff
@@ -1,5 +1,5 @@
 class AddDefaultValueForOrderStatus < ActiveRecord::Migration
   def change
-    change_column_default(:orders, :current_status, "completed")
+    change_column_default(:orders, :current_status, 'completed')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.